subscription/notify-test-lambda/requirements.txt

```diff
@@ -0,0 +1 @@
+boto3
```

Comment:
no trailing new lines? The one above does have one.

Reply:
I added a trailing new line.